pkg/machine/ignition.go

```diff
@@ -307,7 +307,12 @@ machine_enabled=true
 	delegateConf := `[Service]
 Delegate=memory pids cpu io
 `
-	subUID := `%s:100000:1000000`
+	// Prevent subUID from clashing with actual UID
+	subUID := 100000
+	if uid >= 100000 && uid < 1100000 {
```

Comment:
Suggest:
        if uid >= subUID && uid < (subUID+100000) {

You might want to make the 100000 become a constant as well
subUIDs=100000
        if uid >= subUID && uid < (subUID+subUIDs) {